mne/utils/numerics.py

```diff
@@ -1152,3 +1152,9 @@ def cache_fun(*args):
         return cache_fun
 
     return dec
+
+
+def _array_repr(x):
+    """Produce compact info about float ndarray x."""
+    assert isinstance(x, np.ndarray), type(x)
```

Comment:
I think bandit will complain about `assert` appearing outside a test file (but maybe not?)

Reply:
never mind, bandit does complain but it's considered "low" severity so the job still passes. And I don't mind having this kind of assert usage in our code anyway.

Reply:
I think we ignore that rule because we do this so often